modules/ROOT/pages/design-modify-raml-specs-conform.adoc

```diff
@@ -0,0 +1,791 @@
+= Modify Published RAML API Specifications to Conform Completely to RAML 0.8 or 1.0
+
+[[bookmark-a,Back to the top]]
+
+As of November 1, 2018, the API editor Design Center is now more precise in ensuring that the RAML in your API specifications is valid. However, it is possible that one or more of the APIs that you used previous versions of the API editor to create and publish to Anypoint Exchange includes invalid RAML that those previous versions did not flag.
+
+Check your published APIs by opening them in the API editor in Design Center. You can use the guide on this page to correct a number of errors that MuleSoft has found while testing the current API editor on a large number of RAML API specifications. If you see errors that this guide does not include, refer to the specifications for RAML 0.8 and 1.0, which you can read on https://raml.org/[raml.org].
```

Comment:
We started short-handing "RAML API specification" for just "RAML" in the next paragraph. Should be consistent here.

```suggestion
Check your published APIs by opening them in the API editor in Design Center. You can use the guide on this page to correct a number of errors that MuleSoft has found while testing the current API editor on a large number of RAML. If you see errors that this guide does not include, refer to the specifications for RAML 0.8 and 1.0, which you can read on https://raml.org/[raml.org].
```

Reply:
"a large number of RAML" doesn't make sense here.